server/package.json

```diff
@@ -1,4 +1,5 @@
 {
+  "private": true,
```

Comment:
Prevents an accidental push to npm, although that probably isn't a concern since there is no `name` or `version` field or any other required fields in the package.json file. ¯\\\_(ツ)\_/¯